tests/functional/adapter/materialized_view_tests/utils.py

```diff
@@ -67,3 +68,20 @@ def query_autorefresh(project, relation: RedshiftRelation) -> bool:
         and trim(mv.db_name) ilike '{ relation.database }'
     """
     return project.run_sql(sql, fetch="one")[0]
+
+
+def run_dbt_and_capture_with_retries(args: List[str], max_retries: int = 10):
+    """
+    We need to retry `run_dbt` calls on Redshift because we get sporadic failures of the form:
+
+        Database Error in model my_materialized_view (models/my_materialized_view.sql)
+        could not open relation with OID 14957412
+    """
+    retries = 0
+    while retries < max_retries:
+        try:
+            # there's no point to using this with expect_pass=False
```

Comment:
should we add a `sleep` here? Just thinking if we run these in parallel are we just going to be spamming the DWH

Reply:
The tests run in parallel, but this loop should be in sequence. Would adding sleep do anything to solve for the parallel tests?

Reply:
...sigh not really